server/app/views/admin/programs/ProgramImageView.java

```diff
@@ -81,6 +119,70 @@ private FormTag createImageDescriptionForm(
                 .setLabelText("Image description")
                 .setValue(form.value().get().getSummaryImageDescription())
                 .getInputTag())
-        .with(submitButton("Save description").withClass(ButtonStyles.SOLID_BLUE));
+        .with(
+            submitButton("Save description")
+                .withForm(IMAGE_DESCRIPTION_FORM_ID)
+                .withClass(ButtonStyles.SOLID_BLUE));
+  }
+
+  private DivTag createImageUploadForm(ProgramDefinition program) {
+    StorageUploadRequest storageUploadRequest = createStorageUploadRequest(program);
+    FormTag form =
+        fileUploadViewStrategy
+            .renderFileUploadFormElement(storageUploadRequest)
+            .withId(IMAGE_FILE_UPLOAD_FORM_ID);
+    ImmutableList<InputTag> fileUploadFormInputs =
+        fileUploadViewStrategy.fileUploadFormInputs(
+            Optional.of(storageUploadRequest),
+            MIME_TYPES_IMAGES,
+            fileInputId,
+            /* ariaDescribedByIds= */ ImmutableList.of(),
```

Comment:
I thiiink this `ariaDescribedByIds` field is actually used on the applicant side when answering the file upload question in order to have any file upload form errors be part of the aria label for the `<input>` on the `<form>` -- I'm basing that on https://github.com/civiform/civiform/blob/b4920c4b78022027aab046597c012967c30efe91/server/app/views/questiontypes/FileUploadQuestionRenderer.java#L70-L72, whose call stack eventually goes to https://github.com/civiform/civiform/blob/b4920c4b78022027aab046597c012967c30efe91/server/app/views/questiontypes/ApplicantQuestionRendererImpl.java#L107-L108. I'm not 100% sure about that since I don't have a ton of experience with the applicant UI. But, I think this `airaDescribedByIds` would just provide extra information to screen readers about what the `<input>` part of the form is for, and wouldn't let the admin set anything additional in the form itself.

Reply:
Ohhh that makes sense! I see an alt-text box in the design so that will be handled :)